common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_apmanager.yaml

```diff
@@ -184,6 +184,23 @@ cACTION_APMANAGER_STEERING_EVENT_AUTH_FAIL_NOTIFICATION:
   _type: class
   params: sSteeringEvAuthFail
 
+cACTION_APMANAGER_WIFI_CREDENTIALS_SET_REQUEST:
+  _type: class
+  ssid:
+    _type: char    
+    _length: [ "beerocks::message::WIFI_SSID_MAX_LENGTH" ]
+  pass:
+    _type: char    
+    _length: [ "beerocks::message::WIFI_PASS_MAX_LENGTH" ]
+  auth_type:
+    _type: char    
+    _length: [ "beerocks::message::WIFI_SECURITY_TYPE_MAX_LENGTH" ]
+  enc_type:
+    _type: char    
+    _length: [ "beerocks::message::WIFI_SECURITY_TYPE_MAX_LENGTH" ]
+  is_backhaul: uint8_t
```

Comment:
A BSS has 3 possible modes: fronthaul, backhaul, or backhaul+fronthaul combined. So this should be either:
- a bitmap with fronthaul and backhaul bits;
- an enum with 3 values;
- two boolean fields.

For the first two options, the name is_backhaul is not appropriate. In hostapd, it's an enum with 4 values (because there's also the "not EasyMesh" case), and it's called "multi_ap".

Reply:
Why not using `eWscVendorExtsubelementBssType`? This will also include the teardown.

Reply:
Changed to list of configdata.